packages/theme/src/theme/_navs.scss

```diff
@@ -73,41 +73,42 @@
 	}
 
 	&-tabs {
+		background-color: tint($brand-primary, 90);
```

Comment:
This means we need 2 new shades of $brand-primary : 
* tab background
* tab active background
We already have $brand-primary-light, $brand-primary-lighter. If we come up with 2 new names that express that they are more light, ok for me.